### ISSUE TITLE
0.1100.3 template inspection not working

### ISSUE THREAD

**Issue by nick-lvov-dev** [angular/vscode-ng-language-service#1063]
**Describe the bug**

Basically the title, the template inspection doesn't work, you can't navigate to a variable, errors aren't shown etc.
When I reverted back to 0.1100.2 it started working again

**To Reproduce**

1. Create new angular project
2. Add an object property
3. Try to access a non-existent property of that object from the template

**Expected behavior**

You can navigate to the property definition by pressing F12 or Ctrl + left click
An error is displayed about the missing property

**Actual behavior**
Navigation doesn't work
No error is displayed

**Logs**

Please attach two logs:

1.
```
[0115/133555.049:ERROR:registration_protocol_win.cc(103)] CreateFile: The system cannot find the file specified. (0x2)
[Info  - 1:35:55 PM] Angular language server process ID: 8152
[Info  - 1:35:55 PM] Using typescript/lib/tsserverlibrary v4.0.5 from c:\Users\nikit\.vscode\extensions\angular.ng-template-0.1100.3\node_modules\typescript\lib\tsserverlibrary.js
[Info  - 1:35:55 PM] Using @angular/language-service v11.0.9 from c:\Users\nikit\.vscode\extensions\angular.ng-template-0.1100.3\server\node_modules\@angular\language-service\bundles\language-service.js
[Info  - 1:35:55 PM] Log file: c:\Users\nikit\AppData\Roaming\Code\logs\20210115T133438\exthost3\Angular.ng-template\nglangsvc.log
[Info  - 1:35:57 PM] Enabling VE language service for d:/repos/test-angular/tsconfig.json.
[Error - 1:35:57 PM] Failed to find project for d:\repos\test-angular\src\app\app.component.html
[Error - 1:36:11 PM] No config file for d:/repos/test-angular/src/app/app.component.html
```

2. [nglangsvc.log](https://github.com/angular/vscode-ng-language-service/files/5818864/nglangsvc.log)

**Screenshots**

![image](https://user-images.githubusercontent.com/15242942/104690475-13af3780-5737-11eb-9ecb-69de578a6de6.png)
![image](https://user-images.githubusercontent.com/15242942/104690491-1ad64580-5737-11eb-98a8-b227f768e5d6.png)

**Comment by Max-Xtremeapp** [comment]
I have a same problem.

**Comment by bubliq** [comment]
Same problem with 0.1100.3
Downgrade to version v0.1100.2 helped

**Comment by gn131131** [comment]
> Same problem with 0.1100.3
> Downgrade to version v0.1100.2 helped

reinstall vscode and extentions not working, 2h wasted
thankyou

**Comment by bubliq** [comment]
> > Same problem with 0.1100.3
> > Downgrade to version v0.1100.2 helped
> 
> reinstall vscode and extensions not working, 2h wasted
> thankyou

![image](https://user-images.githubusercontent.com/5256688/104730188-a4434300-574a-11eb-82e1-6a69e0c77acb.png)

**Comment by Ruud-cb** [comment]
> > Same problem with 0.1100.3
> > Downgrade to version v0.1100.2 helped
> 
> reinstall vscode and extensions not working, 2h wasted
> thankyou

2 hours wasted? It literally takes up to 1 minute to install another version like bubliq is already showing.
Thanks for the hint, worked for me as well!

**Comment by crclz** [comment]
I haven't write angular apps for 7 month and today I'm about to work on a new project, angular extension gave me this surprise. 

Downgrading to v0.1100.2 solves the problem.

**Comment by gw-casco** [comment]
Same issue. Downgrading helps. 
Fix 🙏.

**Comment by kyliau** [comment]
Thanks for submitting the report. Since there is [no way to roll back](https://github.com/microsoft/vscode-vsce/issues/176) the release, I'll republish 0.1100.2 as the latest version while we investigate the issue.

On the other hand, looking at the change log, I've narrowed the issue down to two commits:

1. https://github.com/angular/angular/pull/39898
2. https://github.com/angular/angular/pull/39917

I suspect this is a Windows-only issue. From the log provided (thanks for uploading), the path separator is inconsistent:
```
[Info  - 1:35:57 PM] Enabling VE language service for d:/repos/test-angular/tsconfig.json.
[Error - 1:35:57 PM] Failed to find project for d:\repos\test-angular\src\app\app.component.html
[Error - 1:36:11 PM] No config file for d:/repos/test-angular/src/app/app.component.html
```

**Comment by kyliau** [comment]
Fixed in v11.1.0, published to the marketplace
https://github.com/angular/vscode-ng-language-service/releases/tag/v11.1.0

**Comment by bubliq** [comment]
> Fixed in v11.1.0, published to the marketplace
> https://github.com/angular/vscode-ng-language-service/releases/tag/v11.1.0

Worked for me, thanks, @kyliau
